jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -74,6 +74,7 @@ public class ErrorHandler implements Request.Handler
 
     boolean _showStacks = true;
     boolean _showMessageInTitle = true;
+    Type _defaultResponseMimeType = Type.TEXT_HTML;
```

Comment:
Please use `String`, so that it will be easier to configure also from XML.

Reply:
Type is changed to String